src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryAddress.java

```diff
@@ -41,8 +41,8 @@
  * <p>
  * All implementations of this interface must be <a href="{@docRoot}/java.base/java/lang/doc-files/ValueBased.html">value-based</a>;
  * programmers should treat instances that are {@linkplain #equals(Object) equal} as interchangeable and should not
- * use instances for synchronization, or unpredictable behavior may occur. The {@code equals} method should
- * be used for comparisons.
+ * use instances for synchronization, or unpredictable behavior may occur. For example, in a future release, synchronization may fail.
```

Comment:
Nit: this line is very long and maybe good to wrap this line and the line above..   I understand that the javadoc in this file has long lines and href is an outliner.

Reply:
Yeah, it's not conventional, but it's the style in this file, so I decided not to perturb it. This is an incubator module, I assume it will get polished up before becoming final.

Reply:
Ok, Roger said the same thing, I guess I tweak it.